mix.exs

```diff
@@ -226,7 +226,10 @@ defmodule Logflare.Mixfile do
       {:opentelemetry_exporter, "~> 1.6"},
       {:opentelemetry_phoenix, "~> 1.1"},
       {:opentelemetry_cowboy, "~> 0.2"},
-      {:live_monaco_editor, "~> 0.1"}
+      {:live_monaco_editor, "~> 0.1"},
+
+      # IP address parsing
+      {:net_address, "~> 0.3.1"}
```

Comment:
dependency not necessary, can use :inet module
https://www.erlang.org/doc/apps/kernel/inet.html#parse_address/1

Reply:
Done!